classes/Pods.php

```diff
@@ -1717,7 +1717,7 @@ public function field( $name, $single = null, $raw = false ) {
 					$post_ID = $this->id();
 				}
 
-				$filter = pods_v( 'display_filter', $field_data );
+				$filter = pods_v( 'display_filter', $field_data, '' );
```

Comment:
```suggestion
				$filter = pods_v( 'display_filter', $field_data, '', true );
```
This will ensure it always ends up the blank string if it's empty-ish